server/sonar-process-monitor/src/main/java/org/sonar/application/process/AbstractProcessMonitor.java

```diff
@@ -24,80 +24,61 @@
 import java.io.InputStream;
 import java.util.concurrent.TimeUnit;
 import javax.annotation.Nullable;
-import org.sonar.process.ProcessCommands;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.sonar.process.ProcessId;
 
-import static java.util.Objects.requireNonNull;
+abstract class AbstractProcessMonitor implements ProcessMonitor {
 
-class ProcessMonitorImpl implements ProcessMonitor {
+  private static final Logger LOG = LoggerFactory.getLogger(EsProcessMonitor.class);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove this reference to "EsProcessMonitor". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2390)